### PR TITLE
Adding a rubygem compiler so we can get an asciidoc binary, and remov…

### PR DESCRIPTION
**Pull request by etsauer**
…e the ruby runtime from hugo image

#### What is this PR About?
see above.

#### How should we test or review this PR?
Follow container-images/hugo/README.adoc

#### Is there a relevant Trello card or Github issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----

- [x] Have you followed the [contributing guidelines](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the uncontained.io guides?

-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: container-images/hugo/README.adoc [hunk 1]
```diff
@@ -4,6 +4,32 @@ This container image is provided to help test and build this hugo site.
...
+
+1. Build the rubygem-compiler
+2. Use the rubygem-compiler image to build an asciidoctor binary
+3. Build the hugo image
+
+Build the rubygem-compiler image:
+
+[source,bash]
+----
+docker build -t redhatcop/rubgem-compiler ./container-images/rubygem-compiler
```
Comment (sabre1041) [inline container-images/hugo/README.adoc]: Spelling error in tag name (rubgem -> rubygem). Should be `redhatcop/rubygem-compiler`

File: container-images/hugo/README.adoc [hunk 2]
```diff
@@ -4,6 +4,32 @@ This container image is provided to help test and build this hugo site.
...
+----
+
+Compile asciidoctor.
+
+[source,bash]
+----
+docker run -it -v ${PWD}/container-images/hugo/root/usr/bin/:/tmp/output/ \
+  -e PACKER_OPTIONS="--gem=asciidoctor --gem-version=1.5.7.1" \
+  -e PACKER_ENTRANCE="asciidoctor" \
+  -e PACKER_OUTPUT_PATH:=/tmp/output/asciidoctor \
```
Comment (sabre1041) [inline container-images/hugo/README.adoc]: Remove the extra `:`

File: container-images/hugo/README.adoc [hunk 3]
```diff
@@ -4,6 +4,32 @@ This container image is provided to help test and build this hugo site.
...
+  redhatcop/rubygem-compiler
+----
+
```
Comment (sabre1041) [inline container-images/hugo/README.adoc]: May want to add some discussion that the `asciidoctor` binary will be created at `container-images/hugo/root/usr/bin/asciidoctor` and used as part of hugo container build

File: container-images/hugo/README.adoc [hunk 4]
```diff
@@ -4,6 +4,32 @@ This container image is provided to help test and build this hugo site.
...
+Build the image.
+
 [source,bash]
 ----
 cd uncontained.io/
```
Comment (sabre1041) [inline container-images/hugo/README.adoc]: This is no longer needed as the user is already in the directory

**Comment by etsauer** [comment]
@sabre1041 done.

**Pull request merged**